jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -634,14 +636,26 @@ public MethodHandle findStatic(Class<?> clazz, String methodName, MethodType typ
 		 * @param methodName - the name of the method
 		 * @param type - the type of the method
 		 * @return a MethodHandle that will do virtual dispatch on the first argument
-		 * @throws IllegalAccessException - if method is static or access is refused
+		 * @throws IllegalAccessException - if method is static or access is refused or clazz's package is not exported to the current module 
```

Comment:
This javadoc will be generated for both 8 & 9+.  Module should only be mentioned for 9

Reply:
Will fix.